integrations/weaviate/src/haystack_integrations/components/retrievers/weaviate/hybrid_retriever.py

```diff
@@ -174,3 +184,73 @@ def run(
             max_vector_distance=max_vector_distance,
         )
         return {"documents": documents}
+
+    @component.output_types(documents=list[Document])
+    async def run_async(
+        self,
+        query: str,
+        query_embedding: list[float],
+        filters: Optional[dict[str, Any]] = None,
+        top_k: Optional[int] = None,
+        alpha: Optional[float] = None,
+        max_vector_distance: Optional[float] = None,
+    ) -> dict[str, list[Document]]:
+        """
+        Asynchronously retrieves documents from Weaviate using hybrid search.
+
+        :param query:
+            The query text.
+        :param query_embedding:
+            Embedding of the query.
+        :param filters: Filters applied to the retrieved Documents. The way runtime filters are applied depends on
+                        the `filter_policy` chosen at retriever initialization. See init method docstring for more
+                        details.
+        :param top_k:
+            The maximum number of documents to return.
+        :param alpha:
+            Blending factor for hybrid retrieval in Weaviate. Must be in the range ``[0.0, 1.0]``.
```

Comment:
let's use single backticks for inline code
```suggestion
            Blending factor for hybrid retrieval in Weaviate. Must be in the range `[0.0, 1.0]`.
```